cranelift/isle/docs/language-reference.md

```diff
@@ -262,19 +262,19 @@ subterms:
 * `_` is a wildcard and matches anything, without capturing it.
 
 * Primitive constant values, such as `42` or `$Symbol`, match only if
-  the term is exactly equal to this constant.
+  the term is an exact equivalent.
 
-These pattern-matching operators can be combined, so we could write,
+These pattern-matching operators can be combined, so we can write,
 for example, `(A (B x _) z)`. This pattern would match the term `(A (B
 1 2) 3)` but not `(A (C 4 5) 6)`.
 
-A pattern can properly be seen as a partial function from input term
-to captured (bound) variables: it either matches or it doesn't, and if
+A pattern can probably be seen as a partial function from the input term
```

Comment:
One cannot replace "properly" with "probably" -- that completely changes the meaning.